doc/nrf/app_dev/device_guides/nrf54l/vpr_flpr.rst

```diff
@@ -0,0 +1,36 @@
+.. _vpr_flpr_nrf54l:
+
+Working with the FLPR core
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF54L15 SoC has a dedicated VPR CPU (RISC-V architecture), named *fast lightweight peripheral processor* (FLPR).
+The following peripherals are available for use with the FLPR core, and can be accessed through the appropriate Zephyr Device Driver API:
+
+* GPIO
+* GPIOTE
+* GRTC
+* TWIM
+* UARTE
+* VPR
```

Comment:
Can you add a short section here to explain how VPR is used to start FLPR? I mean the launcher thing, but I refuse to call it like this. :)

Reply:
added